usr/src/uts/armv8/io/simple-bus.c

```diff
@@ -476,27 +492,34 @@ smpl_intr_ops(dev_info_t *pdip, dev_info_t *rdip, ddi_intr_op_t intr_op,
 
 			int *irupts_prop;
 			int irupts_len;
-			if (ddi_getlongprop(DDI_DEV_T_ANY, rdip, DDI_PROP_DONTPASS, "interrupts", (caddr_t)&irupts_prop, &irupts_len) != DDI_SUCCESS || irupts_len == 0) {
+			if (ddi_getlongprop(DDI_DEV_T_ANY, rdip,
+			    DDI_PROP_DONTPASS, "interrupts",
+			    (caddr_t)&irupts_prop,
+			    &irupts_len) != DDI_SUCCESS ||
+			    irupts_len == 0) {
 				return (DDI_FAILURE);
 			}
-			if (interrupt_cells * hdlp->ih_inum >= irupts_len * sizeof(int)) {
+			if ((interrupt_cells * hdlp->ih_inum) >=
+			    (irupts_len * sizeof (int))) {
+				/* XXX: Size looks wrong */
```

Comment:
`irupts_len`, or the `sizeof (int)`?

Reply:
That was a comment lingering from when I was working out why we treated `irupts_len` as if it were in cells here, but then also passed it to `kmem_free`. 

It's dangling, sorry.

Reply:
Got it - and I saw it get cleaned up later.